app/models/user.rb

```diff
@@ -38,6 +38,7 @@ class User < ActiveRecord::Base
   has_many :permissions,       :through => :filters
   has_many :cached_usergroup_members
   has_many :widgets, :dependent => :destroy
+  has_many :ssh_keys, :dependent => :destroy
```

Comment:
Hm, I can't think of any. The tricky thing is that sometimes the owner is user, sometimes it's user group.